test/packages/good/data_stream/foo/fields/some_fields.yml

```diff
@@ -27,6 +27,11 @@
   description: A field with a pattern defined
   pattern: '^[a-zA-Z]$'
   example: abcd
+- name: foobar
+  type: keyword
+  ignore_above: 5
+  description: A field that can be ignored without failing tests
+  accept_ignored_values: true
```

Comment:
Brainstorming around a better name:

* Something like skip or ignore, not validate
* Something around errors?
* Unfortunately already called ignore 

`skip_ignore_above`, `skip_ignore_above_error`? -> I made it specific as I expect in the future we can tell apart the different reasons. Maybe for now `skip_ignore_error(s)`

Did you used `ignored` on purpose vs `ignore`?